images/ami_kms_key.tf

```diff
@@ -116,11 +116,15 @@ data "aws_iam_policy_document" "ami_kms_doc" {
       # playground, and the Shared Services account.  Any accounts
       # that need to launch EC2 instances from AMIs encrypted using
       # our key should be listed here.
-      values = [
+      values = concat([
         for account in data.aws_organizations_organization.cool.accounts :
         "arn:aws:iam::${account.id}:role/ProvisionAccount"
         if length(regexall("^env[0-9]* \\(${local.this_account_type}\\)$|^Playground Legacy \\(${local.this_account_type}\\)$|^Shared Services \\(${local.this_account_type}\\)$", account.name)) > 0
-      ]
+        ], [
+        for account in data.aws_organizations_organization.cool.accounts :
+        "arn:aws:iam::${account.id}:role/Terraformer"
+        if length(regexall("^env[0-9]* \\(${local.this_account_type}\\)$", account.name)) > 0
+      ])
```

Comment:
The comment above this block should be updated to mention the new Terraformer role.

Reply:
See commit fd7cc0f1b792ac7f191da69ad3f92dfc793dbcae.